ansible/group_vars/all/piecewise_global_config.yml

```diff
@@ -15,7 +15,7 @@ project_name: piecewise
 
 # Information about the fork and branch to deploy.
 # Specify any valid git remote (typically a github username)
-remote: https://github.com/opentechinstitute
+remote: https://github.com/critzo
```

Comment:
Yeah, sorry I didn't mention this. I guess I should just have a long extra_vars list when I'm testing these in order to override the defaults. I'll swap them back in the next PR.